app/src/main/kotlin/com/intive/picover/parties/view/PartiesScreen.kt

```diff
@@ -13,39 +12,49 @@ import androidx.compose.foundation.lazy.items
 import androidx.compose.material.icons.Icons
 import androidx.compose.material.icons.filled.Add
 import androidx.compose.material3.Card
+import androidx.compose.material3.ExperimentalMaterial3Api
 import androidx.compose.material3.FloatingActionButton
 import androidx.compose.material3.Icon
 import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.ModalBottomSheet
 import androidx.compose.material3.Text
+import androidx.compose.material3.rememberModalBottomSheetState
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.LaunchedEffect
 import androidx.compose.runtime.getValue
+import androidx.compose.runtime.rememberCoroutineScope
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.text.style.TextOverflow
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import androidx.navigation.NavHostController
+import com.intive.picover.R
 import com.intive.picover.common.error.PicoverGenericError
 import com.intive.picover.common.loader.PicoverLoader
+import com.intive.picover.common.validator.ValidationStatus
 import com.intive.picover.main.theme.Typography
 import com.intive.picover.parties.model.PartiesEvent
-import com.intive.picover.parties.model.PartiesSideEffect.NavigateToPartyDetails
+import com.intive.picover.parties.model.PartiesSideEffect
 import com.intive.picover.parties.model.Party
 import com.intive.picover.parties.viewmodel.PartiesViewModel
 import kotlinx.coroutines.flow.collectLatest
+import kotlinx.coroutines.launch
 
+@OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun PartiesScreen(
 	viewModel: PartiesViewModel,
 	navController: NavHostController,
 ) {
 	val state by viewModel.state
+	val sheetState = rememberModalBottomSheetState()
+	val coroutineScope = rememberCoroutineScope()
 	LaunchedEffect(true) {
 		viewModel.sideEffects.collectLatest { effect ->
 			when (effect) {
-				is NavigateToPartyDetails -> navController.navigate("partyDetails/${effect.partyId}")
+				is PartiesSideEffect.NavigateToPartyDetails -> navController.navigate("partyDetails/${effect.partyId}")
```

Comment:
why you removed import, is the whole name needed?

Reply:
It happened after the rebase. @bartek977 please see https://github.com/Android-Guild/Picover/pull/320/files

Reply:
@bartek977 Please advise.

Reply:
resolved as it is fixed in #320